docs/proposals/parquet-storage.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Parquet-based Storage"
+linkTitle: "Parquet-based Storage"
+weight: 1
+slug: parquet-storage
+---
+
+- Author: [Ben Ye](https://github.com/yeya24), [Alan Protasio](https://github.com/alanprot)
+- Date: April 2025
+- Status: Proposed
+
+## Background
+
+Since the introduction of Block Storage in Cortex, TSDB format and Store Gateway is the de-facto way to query long term data on object storage. However, it presents several significant chanllenges:
```

Comment:
```suggestion
Since the introduction of Block Storage in Cortex, TSDB format and Store Gateway is the de-facto way to query long term data on object storage. However, it presents several significant challenges:
```